server/src/main/scala/drt/server/feeds/cirium/CiriumFeed.scala

```diff
@@ -99,7 +99,9 @@ object CiriumFeed {
       if (scheduledToNearest5Mins == carrierScheduledTime)
         None
       else
-        Option(carrierScheduledTime)
+        Option(carrierScheduledTime),
+      None,
+      Some(f.departureDate).map(_.millis)
```

Comment:
I'm sure you've already checked this, but is the departure date coming from our cirium feed definitely in UTC?

Reply:
It seems to UTC :-)